pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -175,6 +175,42 @@ def test_to_datetime_format_weeks(self):
 
 class TestToDatetime(object):
 
+    def test_to_datetime_iso_week_year_format(self):
+        data = [
```

Comment:
Let's parametrize this using `pytest`:

~~~python
@pytest.mark.parametrize(...)
def test_to_datetime_iso_week_year_format(s, _format, dt):
...
~~~

Also note that I use "_format" instead of "format" because as a personal preference, I prefer not to use keywords when creating local variables.